conformance/conformance.go

```diff
@@ -116,7 +116,7 @@ func RunConformanceWithOptions(t *testing.T, opts suite.ConformanceOptions) {
 	cSuite, err := suite.NewConformanceTestSuite(opts)
 	require.NoError(t, err, "error initializing conformance suite")
 
-	cSuite.Setup(t)
+	cSuite.Setup(t, tests.ConformanceTests)
 	err = cSuite.Run(t, tests.ConformanceTests)
```

Comment:
It might make sense to collapse setup/run into the same method. Though I felt this was out of scope for this PR.